scripts/start_quickstart_argilla.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Generating hashed password
+hashed_password=$(htpasswd -nbB "" "$USER_PASSWORD" | cut -d ":" -f 2 | tr -d "\n")
+
+# Creating users.yml file
+cat >users.yml <<EOF
+- username: "admin"
+  api_key: $USER_API_KEY
```

Comment:
We need 2 different credentials configuration:

1. The admin username, should be private to quickstart deployers
2. The argilla username that can be shared to login with.

We should have something like:

`ADMIN_PASSWORD`
`ADMIN_API_KEY`
`ARGILLA_PASSWORD`
`ARGILLA_API_KEY`